flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -200,12 +202,13 @@ ScanContext copyWithAppendsBetween(long newStartSnapshotId, long newEndSnapshotI
         .project(schema)
         .filters(filters)
         .limit(limit)
-        .exposeLocality(exposeLocality)
         .includeColumnStats(includeColumnStats)
+        .exposeLocality(exposeLocality)
+        .planParallelism(planParallelism)
```

Comment:
Thanks for catching this, i think we can move this to another separated hot fix commit?

Reply:
i created a hot fix for it #4341